app/admin/jobs/routine/[id]/execution-history.tsx

```diff
@@ -21,18 +21,21 @@ interface ExecutionHistoryProps {
 interface ExecutionHistoryResponse {
   executions: RoutineJobExecution[];
   totalCount: number;
+  availableEnvs: string[];
 }
 
 export default function ExecutionHistory({
   routineJobId,
 }: ExecutionHistoryProps) {
   const [executions, setExecutions] = useState<RoutineJobExecution[]>([]);
   const [totalCount, setTotalCount] = useState(0);
+  const [availableEnvs, setAvailableEnvs] = useState<string[]>([]);
```

Comment:
We don't want to gather `availableEnvs`. Change the filter from being a dropdown to a free text field